### PR TITLE
website: Update all Learn crosslinks

### PR DESCRIPTION
**Pull request by nfagerlund**
The URL structure on Learn recently changed, so it's time to update some URLs.

### CODE REVIEW

File: website/intro/examples/index.html.markdown [hunk 1]
```diff
@@ -25,7 +25,7 @@ Experimenting in this way can help you learn how the Terraform lifecycle
 works, as well as how to repeatedly create and destroy infrastructure.
 
 If you're completely new to Terraform, we recommend reading the
-[getting started guide](https://learn.hashicorp.com/terraform/getting-started/install) before diving into
+[getting started guide](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) before diving into
```
Comment (im2nguyen) [inline website/intro/examples/index.html.markdown:28]:
```suggestion
[Terraform: Get Started](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn before diving into
```

File: website/intro/index.html.markdown [hunk 2]
```diff
@@ -75,5 +75,5 @@ See the page on [Terraform use cases](/intro/use-cases.html) to see the
 multiple ways Terraform can be used. Then see
 [how Terraform compares to other software](/intro/vs/index.html)
 to see how it fits into your existing infrastructure. Finally, continue onwards with
-the [getting started guide](https://learn.hashicorp.com/terraform/getting-started/install) to use
+the [getting started guide](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) to use
```
Comment (im2nguyen) [inline website/intro/index.html.markdown:78]:
```suggestion
the [Terraform: Get Started](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn to use
```

**Review by im2nguyen — APPROVED** [review]
Overall looks good, a couple nitpicky comments but pre-approving so I don't hold you up!

File: website/docs/configuration/provider-requirements.html.md [hunk 3]
```diff
@@ -324,8 +324,8 @@ compatible with Terraform v0.11 or later and should never be declared in a
 ## In-house Providers
 
 Anyone can develop and distribute their own Terraform providers. (See
-the [Call APIs with Terraform Providers](https://learn.hashicorp.com/terraform/providers/provider-use?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS)
-track on HashiCorp Learn for more
+the [Call APIs with Terraform Providers](https://learn.hashicorp.com/collections/terraform/providers?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS)
+collection on HashiCorp Learn for more
 about provider development.)
```
[the remark this answers is not in the document]
Comment (nfagerlund) [reply website/docs/configuration/provider-requirements.html.md:329]: yeah good call

File: website/docs/configuration/provider-requirements.html.md [hunk 4]
```diff
@@ -324,8 +324,8 @@ compatible with Terraform v0.11 or later and should never be declared in a
 ## In-house Providers
 
 Anyone can develop and distribute their own Terraform providers. (See
```
Comment (nfagerlund) [inline website/docs/configuration/provider-requirements.html.md:326]:
```suggestion
Anyone can develop and distribute their own Terraform providers. See
```

File: website/docs/configuration/provider-requirements.html.md (hunk 3 above)
Comment (nfagerlund) [inline website/docs/configuration/provider-requirements.html.md:329]:
```suggestion
about provider development.
```

File: website/docs/provisioners/index.html.markdown [hunk 5]
```diff
@@ -77,7 +77,7 @@ process in various ways data passed via the means described above, allowing
 you to run arbitrary scripts and do basic system configuration immediately
 during the boot process and without the need to access the machine over SSH.
 
-> For a hands-on tutorial about deploying cloud-init scripts in Terraform, try the [Provision Infrastructure with cloud-init](https://learn.hashicorp.com/tutorials/terraform/cloud-init?in=terraform/provision?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) track on HashiCorp Learn.
+> **Hands-on:** Try the [Provision Infrastructure with Cloud-Init](https://learn.hashicorp.com/tutorials/terraform/cloud-init?in=terraform/provision&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
 
 If you are building custom machine images, you can make use of the "user data"
 or "metadata" passed by the above means in whatever way makes sense to your
```
[the remark this answers is not in the document]
Comment (nfagerlund) [reply website/docs/provisioners/index.html.markdown:83]: Oh, good catch. IDK why it didn't have an existing callout down there.

**Pull request merged**